src/measure_theory/set_integral.lean

```diff
@@ -229,6 +229,12 @@ begin
   exact hfi.ae_measurable.null_measurable_set (measurable_set_singleton 0).compl
 end
 
+lemma set_integral_trim {α} {m m0 : measurable_space α} {μ : measure α} (hm : m ≤ m0) {f : α → E}
```

Comment:
same comment as above: I am not sure that you need the integrability assumption.